src/components/MatchForm/MatchFormMobile.tsx

```diff
@@ -55,6 +58,20 @@ export const MatchFormMobile = ({
                 </Text>
               </div>
               <div className={`${classNamePrefix}__match-item__details`}>
+                {sourceLink
+                  ? (
+                    <a
+                      href={sourceLink.href}
+                      target={sourceLink.target}
+                      onClick={e => e.stopPropagation()}
+                      className={`${classNamePrefix}__match-item__link`}
+                    >
+                      <Text size={TextSize.sm} as='span'>
+                        {sourceLink.text}
+                      </Text>
+                    </a>
+                  )
+                  : <div></div>}
```

Comment:
You can do some CSS, but this should be almost same affect as yours, and looks slightly better codewise.
```suggestion
              <HStack>
                {sourceLink && (
                    <a
                      href={sourceLink.href}
                      target={sourceLink.target}
                      onClick={e => e.stopPropagation()}
                      className={`${classNamePrefix}__match-item__link`}
                    >
                      <Text size={TextSize.sm} as='span'>
                        {sourceLink.text}
                      </Text>
                    </a>
                  )}
                </HStack>
```